docs: adding default for azure and aws (#4146)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.1/choose-infrastructure/azure/advanced/prerequisites/index.md

<p class="message--note"><strong>NOTE: </strong>On macOS, Docker runs in a virtual machine. Configure this virtual machine with at least 8GB of memory.</strong></p>

> If you create a cluster on Azure using DKP's default settings without any edits to any configuration files or additional flags, your cluster will be deployed on an [Ubuntu 20.04 operating system image][supported-systems] with 3 control plane nodes and 4 worker nodes.

Reviewer: ```suggestion
If you create a cluster on Azure using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on an [Ubuntu 20.04 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes.
```
